jsettlers.logic/src/main/java/jsettlers/ai/highlevel/AiMapInformation.java

```diff
@@ -37,58 +48,44 @@
  */
 public class AiMapInformation {
 
-	public static final int GRAS = EResourceType.VALUES.length;
+	public static final int GRASS_INDEX = EResourceType.VALUES.length;
 	private static final double FISH_TO_FISHER_HUTS_RATIO = 80F / 1F;
```

Comment:
The aim of the BuildingConstants is to be used anywhere (in the AI) where the ratio between two buildings is needed (like in AiMapInformation and in EconomyMinister).
The remaining constants of the AiMapInformation are for the AiMapInformation only (due to performance issues the GRASS_INDEX is used by AiStatistics to fill the AiMapInformation to not run through the whole map twice)